learnEcl/2300-setof.md

```diff
@@ -1,4 +1,40 @@
 ---
-title: Set of
+title: Set Of
 slug: set-of
 ---
+# SET OF
+
+SET OF is a data type which defines attributes that are a set of data elements. All elements must contain the same data type. Indexing in SET OF start with 1. Since data isn't distributed it has to fit in memory.
```

Comment:
<datatype> must be a scalar data type or, in certain circumstances, datasets.

It is worth saying something like "a SET OF data type is a lot like an array in other languages"